feature/policy_forwarding/otg_tests/match_dscp_indirect_next_hop/README.md

```diff
@@ -0,0 +1,75 @@
+# PF-1.1: IPv4/IPv6 policy forwarding to indirect NH matching DSCP/TC.
+
+## Summary
+
+The test verifies policy forwarding(PF) when matching specific DSCP/traffic-class(TC) in IPv4/IPv6 header and pointing to indirect BGP next-hop.
+2 rightmost DSCP/TC bits are used for this test with all the possibles combinations of 3 left-most DSCP/TC bits.
+
+## Setup
+
+
+*   DUT has an ingress port (Port 1) and 2 egress ports combined.
+
+    ```
+                             |         | --eBGP-- | ATE Port 2 |
+        [ ATE Port 1 ] ----  |   DUT   |          |            |
+                             |         | --eBGP-- | ATE Port 3 |
+    ```
+
+*   Traffic is generated from ATE Port 1.
+*   ATE Port 2 is used as the destination port for PF. eBGP peerings
+        on this port announces BG4IE NH.
+*   ATE Port 3 is used as the fallback destination when PBR NH routes
+        are withdrawn.
+
+### Configuration
+
+1.  All DUT Ports are configured as a singleton IP interfaces.
+
+2.  Static routes to test IPv4 and IPv6 destination networks are configured on DUT towards ATE Port 3.
+
+3.  eBGP session is configured on DUT port 2. Indirect /32 and /128 prefixes (PF next-hops) are announced via eBGP.
+
+4.  PF matching traffic marked with rightmost DSCP/TC 2 bits set to `11` is configured on DUT port 1. PF
```

Comment:
Can we rearrange the sentence for clarity?

PF is configured on DUT port 1 to match the traffic marked with  rightmost DSCP/TC 2 bits set to `11`.